spec/controllers/spree/orders_controller_spec.rb

```diff
@@ -224,16 +224,9 @@
           } }
         }
 
-        # Before issuing the update, the second adjustment, which is associated
-        # to the shipment, is already open thus restoring its state leaves it
-        # also open.
-        #
-        # The third adjustment is originated from an EnterpriseFee and it gets
-        # created by #update_distribution_charge! in
-        # app/models/spree/order_decorator.rb:220, which is in turn triggered
-        # by the `contents_changed` notification event defined in
-        # app/models/spree/order_decorator.rb:7
```

Comment:
so all this was caused by the factories?

Reply:
this was not correct, there is no enterprise fee in the order.
the third adjustment was an extra adjustment coming from one of the line items with target_shipment defined in a first stage in order_with_totals_and_distribution.